dealership/customers.py

```diff
@@ -1,11 +1,20 @@
 class Person(object):
     def __init__(self, first_name, last_name, email):
-        pass
+        self.first_name = first_name
+        self.last_name = last_name
+        self.email = email
+    
+    def is_employee(self):
+        raise NotImplementedError()
 
 
 class Customer(Person):
-    pass
+    def is_employee(self):
+        return False
 
 
 class Employee(Person):
-    pass
+    discount = 0.1
```

Comment:
Discount is only defined by Employee. It'd possibly break your interfaces if someone tries to use `interest` from another type of `Person`.
Regarding style: class variables are usually capitalized (`DISCOUNT`).